src/lib/y2network/virtualization_config.rb

```diff
@@ -79,18 +79,19 @@ def create
 
   private
 
-    # Convenience method that returns true if the current item has link and can
-    # be enslaved in a bridge.
+    # Convenience method that returns true if the interface given is connected
+    # and can be added as a bridge port.
     #
-    # @return [Boolean] true if it is bridgeable
+    # @param bridge_builder [Y2Network::InterfaceConfigBuilders::Bridge]
+    # @param interface [Y2Network::Interface] bridge candidate member
+    # @return [Boolean] true if it is connected and bridgeable
     def connected_and_bridgeable?(bridge_builder, interface)
       if !bridge_builder.bridgeable_interfaces.map(&:name).include?(interface.name)
         log.info "The interface #{interface.name} cannot be proposed as bridge."
         return false
       end
 
-      hwinfo = interface.hardware
-      if !hwinfo.present? || !hwinfo.link
+      unless interface.connected?
         log.warn("Lan item #{interface.inspect} has link:false detected")
```

Comment:
lan item is no longer true

Reply:
Ups leftover :P

Reply:
yeah it does not surrender that easily ;-)